<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
border-radius:20px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#fb2525;;
border-style:outset;
border-radius:20px;
}
QPushButton:hover{
border-radius:20px;
background-color:rgb(250, 192, 192);
font: 75 10pt &quot;SansSerif&quot;;
color:rgb(255, 255, 255)

}
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Himalaya</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 50px &quot;Microsoft Himalaya&quot;;
border-radius:20px;
color:white;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 10pt &quot;SansSerif&quot;;
border-radius:10px;
border-bottom:2px solid maroon ;
background: transparent;
    outline: none;
    height: 50px;
    color: whitesmoke;
    font-size: 100%;
padding: 0 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 10pt &quot;SansSerif&quot;;
border-radius:10px;
border-bottom:2px solid maroon ;
background: transparent;
    outline: none;
    height: 50px;
    color: whitesmoke;
    font-size: 100%;
padding: 0 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SansSerif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 10pt &quot;SansSerif&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Full Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SansSerif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 10pt &quot;SansSerif&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Account Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="log">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 63 11pt &quot;Bahnschrift SemiBold SemiConden&quot;;
text-decoration: underline;
color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#fb2525;;
border-radius:20px;
border-style:outset;
}

QPushButton:hover{
background-color:rgb(250, 192, 192);
}
</string>
    </property>
    <property name="text">
     <string>Register for an account</string>
    </property>
   </widget>
   <zorder>label2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label3</zorder>
   <zorder>label4</zorder>
   <zorder>log</zorder>
   <zorder>button2</zorder>
   <zorder>button1</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
